cmd/awscollector/main.go

```diff
@@ -97,6 +97,10 @@ func buildAndParseFlagSet(featgate *featuregate.Registry) (*flag.FlagSet, error)
 	log.Printf("attn: users of the statsd receiver please refer to " +
 		"https://github.com/aws-observability/aws-otel-collector/issues/2249 in regards to an ADOT Collector v0.33.0 " +
 		"breaking change")
+	// TODO: remove after ADOT Collector v0.35.0 is released
+	log.Printf("attn: users of the awscontainerinsightreceiver please refer to " +
+		"https://github.com/aws-observability/aws-otel-collector/issues/**** in regards to an ADOT Collector v0.35.0 " +
```

Comment:
update to link with actual issue when created

Reply:
Updated.